test/parallel/test-preload.js

```diff
@@ -88,16 +88,17 @@ const replProc = childProcess.spawn(
 );
 replProc.stdin.end('.exit\n');
 let replStdout = '';
-replProc.stdout.on('data', function(d) {
+replProc.stdout.on('data', (d) => {
   replStdout += d;
 });
 replProc.on('close', function(code) {
   assert.strictEqual(code, 0);
   const output = [
     'A',
     '> '
-  ].join('\n');
-  assert.strictEqual(replStdout, output);
+  ];
+  assert.ok(replStdout.startsWith(output[0]));
+  assert.ok(replStdout.endsWith(output[1]));
```

Comment:
I have a slight preference for keeping the welcome message at the very top and to include preloaded output afterwards. But I am not sure if that's significantly more work or not. If it is, I am fine to keep it as is. We could still change it at some point.